routemaster/subscription.rb

```diff
@@ -0,0 +1,22 @@
+module Routemaster
+  class Subscription
+
+    attr_reader :subscriber, :callback, :topics, :events
+
+    def initialize(options)
+      @subscriber = options.fetch('subscriber')
+      @callback   = options.fetch('callback')
+      @topics     = options.fetch('topics')
+      @events     = options.fetch('events').symbolize_keys
```

Comment:
Sorry, I wasn't clear. We actually provide `symbolize_keys` with Routemaster, in `core_ext/hash.rb`, but not in `routemaster-client`.

Sidekiq seems to do the same thing in `lib/sidekiq/core_ext.rb`.

I've taken the liberty to add a stub spec for this class to highlight the problem:

```
be rspec spec/subscription_spec.rb

Randomized with seed 39315
F

Failures:

  1) Routemaster::Subscription#initialize passes
     Failure/Error: expect { subject }.not_to raise_error

       expected no Exception, got #<NoMethodError: undefined method `symbolize_keys' for {}:Hash> with backtrace:
         # ./routemaster/subscription.rb:10:in `initialize'
         # ./spec/subscription_spec.rb:13:in `new'
         # ./spec/subscription_spec.rb:13:in `block (3 levels) in <top (required)>'
         # ./spec/subscription_spec.rb:16:in `block (4 levels) in <top (required)>'
         # ./spec/subscription_spec.rb:16:in `block (3 levels) in <top (required)>'
     # ./spec/subscription_spec.rb:16:in `block (3 levels) in <top (required)>'

Finished in 0.01478 seconds (files took 0.37826 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/subscription_spec.rb:15 # Routemaster::Subscription#initialize passes

Randomized with seed 39315
```

So a require is needed in this file — possibly the bit inside Sidekiq since we already depend on it, although that feels ugly as there's a lot of other stuff in that `sidekiq/core_ext.rb` file.